resources/texts/_contentManager.html

```diff
@@ -74,13 +75,16 @@
             downloadUpdaters[book.id] = setInterval(function() { getDownloadInfo(book.id); }, 1000);
           });
         },
-        eraseBook : function(book) {
-            contentManager.eraseBook(book.id);
+        eraseBook : function(id) {
```

Comment:
Maybe rename the function to `eraseBookById`.
There is no type checking in js. `eraseBook` imply to the argument is a book, not an id.
And nothing will warn us if we do something wrong... until the user try to erase the book.